foundation/eclipselink.extension.oracle.spatial.test/antbuild.xml

```diff
@@ -194,308 +199,540 @@
+        <path id="oracle_spatial_test.compile.path">
+            <path refid="oracle_spatial_test.compile.core.path"/>
+            <path refid="oracle_spatial_test.public.dependency.path"/>
+            <path refid="oracle_spatial_test.oracle.dependency.path"/>
+            <path refid="oracle_spatial_test.test.infrastructure.path"/>
+        </path>
+        <path id="oracle_spatial_test.runtime.only.path">
+            <pathelement path="${oracle_spatial_test.build.location}/${oracle.spatial.test.jar}"/>
+            <pathelement path="${oracle_spatial_test.build.location}/${oracle.spatial.jpa.test.jar}"/>
+            <pathelement path="${oracle_spatial_test.build.location}/${eclipselink.struct.converter.model}.jar"/>
+            <pathelement path="${oracle_spatial_test.run.dir}"/>
+            <pathelement path="${jdbc.driver.jar}"/>
+            <pathelement path="${oracle_spatial_test.2.core.test.dir}/resource"/>
         </path>
         <!-- Classpath used for running tests. -->
-        <path id="oracle_test.run.path">
-            <path refid="oracle_test.compile.path"/>
-            <path refid="oracle_test.runtime.only.path"/>
+        <path id="oracle_spatial_test.run.path">
+            <path refid="oracle_spatial_test.compile.path"/>
+            <path refid="oracle_spatial_test.runtime.only.path"/>
         </path>
     </target>
-    <target name="announce_missing_dependencies" unless="dependency.ok" description="publicize missing dependencies">
-        <echo message="Following is a list of dependencies and their status ('found=' anything but 'true' means it wasn't found):"/>
-        <echo message="    Test infrastructure:"/>
-        <echo message="        '${junit.lib}' :: found='${junit.lib.exist}'"/>
-        <echo message="        '${oracle_test.2.jpa.test.dir}/${jpatest.framework.jar}' :: found='${jpatest.framework.exist}'"/>
-        <echo message="        '${oracle_test.2.core.test.dir}/${core.test.jar}' :: found='${core.test.models.exist}'"/>
-        <echo message="        '${oracle_test.2.core.test.dir}/${coretest.framework.jar}' :: found='${test.framework.exist}'"/>
-        <echo message="The test framework is currently part of the core.test and jpa.test projects. Be sure they are built."/>
-        <echo message="    oracle dependency location: '${oracle.extensions.depend.dir}' :: found='${oracle.depend.dir.exist}'"/>
-        <echo message="    dependency location:        '${extensions.depend.dir}' :: found='${ext.depend.dir.exist}'"/>
-        <echo message="        '${jdbc.driver.jar}'        :: found='${jdbc.jar.exist}'"/>
-        <echo message="        '${oracle.sdoapi.lib}'      :: found='${oracle.sdoapi.lib.exist}'"/>
-        <echo message="Please put your Oracle dependencies in one of the two 'dependency' locations (default or custom defined)."/>
-        <fail message="Missing dependencies!" unless="compile.ok"/>
+
+    <target name="init-test-macro" depends="init">
```

Comment:
Those init-something-macro targets are also repeating im many build files. I believe that there is a lot of common code that can be extracted to a single file.